crawlclima/redemet/rmet.py

```diff
@@ -162,14 +175,27 @@ def capture(station, date):
         status = resp.status_code
         time.sleep(wait)
         wait *= 2
-    page = resp.text
-    dataframe = parse_page(page)
-
-    data = describe(dataframe)
-    if len(data) == 0:
-        logger.warning("Empty data for %s", date)
-        return {}
-    data["date"] = date
-    data["station"] = station
-
-    return data
+    # page = resp.text
+    r_data = resp.json()
+
+    for dados in r_data["data"]["data"]:
+        mensagem = dados['mens']
+        date_receive = dados['recebimento']
+
+        # format date
+        date_time_str = datetime.datetime.strptime(
+            date_receive, '%Y-%m-%d %H:%M:%S'
+        )
+        formated_date = date_time_str.strftime('%Y%m%d%H')
+
+        page = formated_date + " - " + mensagem
+
+        dataframe = parse_page(page)
+        data = describe(dataframe)
+        if len(data) == 0:
+            logger.warning("Empty data for %s", date)
+            return {}
+        data["date"] = date
+        data["station"] = station
+
+        return data
```

Comment:
use return inside the `for` loop would be the same as to return just data related to `r_data["data"]["data"][0]`
so maybe it is not the desired behavior. 

my first guess is that your `for` loop should be used just for preparing the variable `page`, for example:



```suggestion
    page = ''
    for dados in r_data["data"]["data"]:
        mensagem = dados['mens']
        date_receive = dados['recebimento']

        # format date
        date_time_str = datetime.datetime.strptime(
            date_receive, '%Y-%m-%d %H:%M:%S'
        )
        formated_date = date_time_str.strftime('%Y%m%d%H')

        page += '{} - {}\n'.format(formated_date, mensagem)

    dataframe = parse_page(page)
    data = describe(dataframe)
    if len(data) == 0:
        logger.warning("Empty data for %s", date)
        return {}
    data["date"] = date
    data["station"] = station

    return data
```

Reply:
This fixed the warning message in len(data) none.